# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoader.h
# Commit: 707a48ef2338
# Full Hash: 707a48ef23385b4b143a9218fe788efcd3e1e3fc
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 15:55:11
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 3 - Remove method to createRemoteFrameLoader from nsIMozBrowserFrame interface. r=farre
#   
#   This appears unused and adds unneeded surface area for these API's to support.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31431
# ==============================================================================

diff -r 37fb118dac6c -r 707a48ef2338 dom/base/nsFrameLoader.h
--- a/dom/base/nsFrameLoader.h	Wed Apr 24 23:05:40 2019 -0500
+++ b/dom/base/nsFrameLoader.h	Wed Apr 24 23:25:56 2019 -0500
@@ -342,7 +342,7 @@
    * this object, which means you can't have called ShowRemoteFrame()
    * or ReallyStartLoading().
    */
-  void SetRemoteBrowser(nsIRemoteTab* aBrowserParent);
+  void InitializeFromBrowserParent(BrowserParent* aBrowserParent);
 
   /**
    * Stashes a detached nsIFrame on the frame loader. We do this when we're